model-optimizer/extensions/middle/ONNXResize11ToInterpolateV4.py

```diff
@@ -0,0 +1,166 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+import numpy as np
+
+from extensions.ops.Cast import Cast
+from extensions.ops.elementwise import Add, Div, Mul
+from extensions.ops.interpolate import Interpolate
+from mo.front.common.layout import get_depth_dim, get_height_dim, get_width_dim
+from mo.front.common.partial_infer.utils import int64_array, float_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.graph.graph import Graph, rename_nodes
+from mo.ops.const import Const
+from mo.ops.shape import Shape
+from mo.ops.strided_slice import StridedSlice
+
+
+class ONNXResize11ToInterpolate4(MiddleReplacementPattern):
+    """
+    The transformation replaces ONNX Resize 11 with Interpolate-3.
+    """
+    enabled = True
+
+    def run_before(self):
+        from extensions.middle.InterpolateSequenceToInterpolate import InterpolateSequenceToInterpolate
+        return [InterpolateSequenceToInterpolate]
+
+    def pattern(self):
```

Comment:
Do not use pattern approach. Instead, use `for resize in graph.get_op_nodes(op='ONNXResize11'):`

Reply:
Fixed.